qoqo_qiskit_devices/Cargo.toml

```diff
@@ -18,13 +18,13 @@ doctest = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-qoqo = { version = "1.4", default-features=false}
-roqoqo = "1.4"
-qoqo-macros = "1.4"
+qoqo = { version = "1.5.1", default-features=false}
+roqoqo = "1.5.1"
+qoqo-macros = "1.5.1"
```

Comment:
Any reason for specifying the last digit? By writing just "1.5" it should automatically use the last available version